schemas/post.py

```
@@ -0,0 +1,52 @@
+from datetime import datetime
+from typing import List
```

Ruff (0.14.11)

`typing.List` is deprecated, use `list` instead

(UP035)